nex/config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Are we planning on checking this file in, or is this an artifact of running nex?